astroquery/desi/tests/test_desi_remote.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import astroquery.desi
```

Comment:
extreme nitpick: Do import the class rather than using the full namespace 
```suggestion
from  astroquery.desi import DESILegacySurvey
```